datacube_utilities/query.py

```diff
@@ -26,8 +27,9 @@ def create_base_query(aoi, res, output_projection, aoi_crs, dask_chunks):
         "output_crs": output_projection,
         "resolution": resolution,
         "dask_chunks": dask_chunks,
-        "crs": aoi_crs,
+        "crs": "EPSG:3460",
```

Comment:
I don't understand why the projection is fixed here. Is not supposed to be passed as an argument?

Reply:
This is the CRS of the AOI. which we have just transformed into EPSG:3460 so that we have a fixed known working CRS for the datacube to work with.